docker/images/fuseki/.dockerignore

```diff
@@ -0,0 +1 @@
+t11
```

Comment:
I would at least comment it out if you want to keep it in. Otherwise it
will exclude the code you want in your image

Freddie Rosario
*DevOps Engineer*
*Based in Florida*

EcoHealth Alliance <http://www.ecohealthalliance.org/>
460 West 34th Street – 17th floor
New York, NY 10001

1.305.851.2925 (mobile)
1.212.380.4465 (fax)

*EcoHealth Alliance leads cutting-edge research into the critical
connections between human and wildlife health and delicate ecosystems. With
this science we develop solutions that promote conservation and prevent
pandemics.*

On Mon, Jun 20, 2016 at 11:19 AM, Stephen Matta <notifications@github.com>
wrote:

> In docker/images/fuseki/.dockerignore
> <https://github.com/ecohealthalliance/infrastructure/pull/28#discussion_r67708701>
> :
>
> > @@ -0,0 +1 @@
> > +t11
>
> Yeah - in the commit message "for future use" but - I can back out that
> commit. I added it for parallelism among the different images but that's
> unnecessary.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ecohealthalliance/infrastructure/pull/28/files/f4fe730598f6fd1bd12d18a1cc2c30910ae6d7b1#r67708701>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAfU9JcxrgqiOamSxwKeZFcoDwOp_-thks5qNq-egaJpZM4I5wTD>
> .
>